packages/react-composites/src/composites/ChatComposite/file-sharing/FileCard.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Icon, mergeStyles, ProgressIndicator, Stack, Text, useTheme } from '@fluentui/react';
+import { getFileTypeIconProps } from '@fluentui/react-file-type-icons';
+import React from 'react';
+
+/**
+ * @beta
+ */
+export interface FileCardProps {
+  /**
+   * File name.
+   */
+  fileName: string;
+  /**
+   * Extension of the file used for rendering the file icon.
+   */
+  fileExtension: string;
+  /**
+   * File upload/download progress percentage between 0 and 1.
+   * Not shown if undefined or 1.
+   */
+  progress?: number;
+  /**
+   * Icon to display for actions like download, upload, etc. on the right of file name.
+   */
+  actionIcon?: JSX.Element;
+  /**
+   * Function that runs when actionIcon is clicked
```

Comment:
Is the action run when `actionIcon` is clicked, or when the `FileCard` is clicked?

Reply:
When actionIcon is clicked.

Reply:
Rewording my question: *Should* the action run when `actionIcon` is clicked, or when the `FileCard` is clicked?

Reply:
For upload, only when the actionIcon is clicked, for download (possibly) when filecard is clicked.